makesig.py

```diff
@@ -105,10 +105,9 @@ def process(start_at = MAKE_SIG_AT['fn']):
 		ins = ins.getNext()
 		
 		if ins.getAddress() != expected_next:
-			# we don't have a good way to deal with alignment bytes
-			# raise an exception for now
-			raise Exception("Instruction at %s is not adjacent"
+			printerr("Instruction at %s is not adjacent"
 					" to previous (expected %s)" % (expected_next, ins.getAddress()))
+			break
```

Comment:
Is this supposed to result in not finding a unique signature? My immediate thought is that it'd be better to fill in discontiguous bytes with wildcards, but in practice I'm not sure.

Reply:
If it's possible to have a chance of getting a signature by wildcarding them until we get out of no-man's-land then that'd be great. May need to have a threshold of number of instructions before we abandon all hope?

Reply:
Wildcarding `ins.GetAddress() - expected_next` bytes may suffice; just peeked and [the `Instruction.getNext()` docs indicate](https://ghidra.re/ghidra_docs/api/ghidra/program/model/listing/Instruction.html#getNext()) that it's the one that follows in address order.

Reply:
That worked for me with some limited testing. Proposed change in https://github.com/data-bomb/makesig/commit/ba7151a0fb1882a0d1af51b212f7e141144234fd. It may not get exercised a lot but it should be helpful. I also tried removing the safeguards of remaining in the same function to confirm it would work in this scenario as well.